### PR TITLE
cifar10 example doc

### PR DESCRIPTION
**Pull request by PurityFan**
Add doc for cifar10_pytorch example

### CODE REVIEW

File: docs/cifar10_examples.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# CIFAR10 examples
+
+The CIFAR-10 dataset is a collection of images that are commonly used to train machine learning and computer vision algorithms. It is one of the most widely used datasets for machine learning research. The CIFAR-10 dataset contains 60,000 32x32 color images in 10 different classes. There are 6,000 images of each class. Thus, we use CIFAR10 as example to introduce using NNI in the field of computer vision.
```
Comment (xuehui1991) [inline docs/cifar10_examples.md]: shall we introduce the search space and tell the user which parameter we want to tuner in this example?

File: docs/cifar10_examples.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+This example requires pytorch. pytorch install package should be chosen based on python version and cuda version.
+
+Here is an example of the environment python==3.5 and cuda == 8.0, then using the following commands to install pytorch:
+
+```bash
+python3 -m pip install http://download.pytorch.org/whl/cu80/torch-0.4.1-cp35-cp35m-linux_x86_64.whl
+python3 -m pip install torchvision
+```
+
+In this experiment, we will search the optimal hyperparamters for **learning rates**, **optimizeation methods** and **network structures**.
```
Comment (xuehui1991) [inline docs/cifar10_examples.md]: optimization

File: docs/cifar10_examples.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
+# CIFAR10 examples
+
+The CIFAR-10 dataset is a collection of images that are commonly used to train machine learning and computer vision algorithms. It is one of the most widely used datasets for machine learning research. The CIFAR-10 dataset contains 60,000 32x32 color images in 10 different classes. There are 6,000 images of each class. Thus, we use CIFAR10 as example to introduce using NNI in the field of computer vision.
+
+**cifar10 with NNI**
+
+This example requires pytorch. pytorch install package should be chosen based on python version and cuda version.
+
+Here is an example of the environment python==3.5 and cuda == 8.0, then using the following commands to install pytorch:
```
Comment (leelaylay) [inline docs/cifar10_examples.md]: We can add a link to https://pytorch.org/ to guide the user to install the pytorch.

File: docs/cifar10_examples.md (hunk 2 above)
Comment (PurityFan) [reply docs/cifar10_examples.md]: Thanks! Done.

File: docs/cifar10_examples.md (hunk 3 above)
Comment (PurityFan) [reply docs/cifar10_examples.md]: Thanks~ Done!

File: docs/cifar10_examples.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+code directory: [examples/trials/cifar10_pytorch/config.yml][6]
+
+Here is the example of running this experiment on OpenPAI:
+
+code directory: [examples/trials/cifar10_pytorch/config_pai.yml][7]
+
+*The complete examples we have implemented: [examples/trials/cifar10_pytorch/][1]*
+
+#### Lauch the experiment
+
```
Comment (xuehui1991) [inline docs/cifar10_examples.md]: I think we could add some picture of result and conclusion here.

Comment (PurityFan) [reply docs/cifar10_examples.md]: Here is the reason for no adding result:
1. If we just start a simple experiment, the final result may be much worse than the current state-of-art result. May cause users to mistakenly think that NNI is not working well.
2. All the docs of the example have not put results on there readme.

**Pull request merged**